src/Taylor1.jl

```diff
@@ -837,6 +845,21 @@ function asin(a::Taylor1)
 end
 
 # Homogeneous coefficients for arcsin
+doc"""
+    asinHomogCoef(kcoef, ac, r_f,coeffs)
+
+Compute the `k-th` expansion coefficient of $c = \arcsin{a}$ given by
+
+\begin{equation*}
+s_k = \frac{1}{ \sqrt{1 - a_0^2} } \big( a_k - \frac{1}{k} \sum_{j=1}^{k-1} j r_{k-j} s_j \big),
+\end{equation*}
+
+with $a$ a `Taylor1` polynomial and $r = \sqrt{1 - a^2}$.
+
+Inputs are the `kcoef`-th coefficient, the vector of the expansion coefficients
+`ac` of `a`, the already calculated expansion coefficients `r_f`
```

Comment:
The coefficients `r_f` are calculated only once. I think better wording is "..., `r_f` the coefficients of $r$ (see above), ...". The same change would be needed for the other functions below.